src/mixin/image.js

```diff
@@ -90,6 +101,10 @@ export default {
     },
 
     defineAspectRatio (img) {
+      // if the aspectRatio property was set, we do not need to detect
+      if (this.aspectRatio) {
```

Comment:
this statement doesn't belong in this method, the only thing this method needs to do is to define the aspect ratio.

Apply those checks in the `loadImage` method like in this screenshot
  
![image 2017-09-29 at 2 06 25 pm](https://user-images.githubusercontent.com/5337152/31015248-6cdd7a9c-a51f-11e7-9f17-b454230cdec8.png)

Reply:
Needs negation like this:
if (this.isPollingKilled && !this.aspectRatio) {